content/ref/cli/wandb-controller.md

```diff
@@ -15,7 +15,7 @@ Run the W&B local sweep controller
 
 | **Option** | **Description** |
 | :--- | :--- |
-| --verbose | Display verbose output |
+| `--verbose` | Display verbose output |
```

Comment:
I'm a little surprised on these `--verbose` flags we don't also support a short flag like `-v`?